concepts/ranges/about.md

```diff
@@ -0,0 +1,132 @@
+# Ranges
+
+[Ranges][ranges] represents an interval between two values.
+The most common types that support ranges are Ints, Chars, and Strings.
+They can be used for many things like quickly creating a collection, slicing strings, checking if a value is in a range and iteration.
```

Comment:
```suggestion
They can be used for many things like quickly creating a collection, slicing strings, checking if a value is in a range, and iteration.
```

This comma is optional and is known as an "Oxford comma". I chose to include it here for consistency because you have used Oxford commas elsewhere.